packages/visitor-keys/src/visitor-keys.ts

```diff
@@ -258,7 +253,9 @@ const additionalKeys: AdditionalKeys = {
   TSTupleType: ['elementTypes'],
   TSTypeAliasDeclaration: ['id', 'typeParameters', 'typeAnnotation'],
   TSTypeAnnotation: ['typeAnnotation'],
-  TSTypeAssertion: SharedVisitorKeys.TypeAssertion,
+  // this is intentionally different to SharedVisitorKeys.AsExpression because
```

Comment:
Personally I don't think this comment is necessary, and would be better omitted (especially considering the previous incarnation of it was wrong). But it's not important one way or another 🤷‍♂️

Reply:
let's remove it